tests/test_tasks.py

```diff
@@ -103,3 +103,69 @@ def test_NoToken(self):
     def test_InvalidToken(self):
         response = self.get(self.VALID_URL, self.bogus_token())
         self.assert_invalid_token(response)
+
+
+class TestTaskPost(EndpointTestCase):
+    """
+    Tests for '/api/students/<int:student_id>/tasks/' [POST] endpoint.
+    """
+
```

Comment:
You can reuse existing implementation of `TestPrizesPost` with prize renamed to task.
This would be preferred, since both Task and Prize have same use case (although in the end task adds points, and prize subtracts them).